components/docs/Card.tsx

```diff
@@ -12,7 +11,7 @@ interface CardProps {
   description: string;
   link: string;
   className: string;
-  Icon: IconType;
+  Icon: React.ComponentType<any>;
```

Comment:
Why are we using any type here?

Reply:
It's throwing an error if i use anything else other than that